packages/contracts/contracts/libs/v0.8.x/minter-libs/SEALib.sol

```diff
@@ -0,0 +1,323 @@
+        return getNextTokenId(_SEAProjectConfig, _projectId);
+    }
+
+    /**
+     * @notice View function that returns the next token ID to be auctioned
+     * by this minter for project `_projectId`.
+     * Reverts if the next token ID has not been populated for the project.
+     * @param _SEAProjectConfig SEAProjectConfig to query
+     * @param _projectId The project ID being queried
+     */
+    function getNextTokenId(
+        SEAProjectConfig storage _SEAProjectConfig,
+        uint256 _projectId
+    ) internal view returns (uint256 nextTokenId) {
+        if (!_SEAProjectConfig.nextTokenNumberIsPopulated) {
+            revert("Next token not populated");
+        }
+        // @dev overflow automatically checked in Solidity ^0.8.0
+        nextTokenId =
+            (_projectId * ONE_MILLION) +
+            _SEAProjectConfig.nextTokenNumber;
+        return nextTokenId;
+    }
+
+    /**
+     * @notice Gets price info to become the leading bidder on a token auction.
+     * If artist has not called `configureFutureAuctions` and there is no
+     * active token auction accepting bids, `isConfigured` will be false, and a
+     * dummy price of zero is assigned to `tokenPriceInWei`.
+     * If there is an active auction accepting bids, `isConfigured` will be
+     * true, and `tokenPriceInWei` will be the sum of the current bid value and
+     * the minimum bid increment due to the minter's
+     * `minterMinBidIncrementPercentage`.
+     * If there is an auction that has ended (no longer accepting bids), but
+     * the project is configured, `isConfigured` will be true, and
+     * `tokenPriceInWei` will be the minimum initial bid price for the next
+     * token auction.
+     * @param _SEAProjectConfig SEAProjectConfig to be evaluated.
+     * @return isConfigured true only if project auctions are configured.
+     * @return tokenPriceInWei price in wei to become the leading bidder on a
+     * token auction.
+     */
+    function getPriceInfoFromSEAProjectConfig(
+        SEAProjectConfig storage _SEAProjectConfig
+    ) internal view returns (bool isConfigured, uint256 tokenPriceInWei) {
+        SEALib.Auction storage _auction = _SEAProjectConfig.activeAuction;
+        // base price of zero not allowed when configuring auctions, so use it
+        // as indicator of whether auctions are configured for the project
+        bool projectIsConfigured_ = projectIsConfigured(_SEAProjectConfig);
+        bool auctionIsAcceptingIncreasingBids_ = auctionIsAcceptingIncreasingBids(
+                _auction
+            );
+        isConfigured =
+            projectIsConfigured_ ||
+            auctionIsAcceptingIncreasingBids_;
+        // only return non-zero price if auction is configured
+        if (isConfigured) {
+            if (auctionIsAcceptingIncreasingBids_) {
+                // return minimum next bid, given current bid
+                tokenPriceInWei = SEALib.getMinimumNextBid(_auction);
```

Comment:
Nit: if this is a library-internal function, can we underscore-case this and not refer to the library in the call (because we're calling from within the library)? 

```suggestion
                tokenPriceInWei = _getMinimumNextBid(_auction);
```

Reply:
re: underscore - we actually call this directly from `MinterSEAV1` as well, so it is a typical library function, so we wouldn't make it private and use a leading underscore
re: not referring to library in the call, good call, and we did that a few places, all of which are updated in [90401a2](https://github.com/ArtBlocks/artblocks-contracts/pull/871/commits/90401a2dec350595832a59a8b2cd3daf92f5d779)

Reply:
AH, missed that. makes sense to me ser – TYVM for the update here.